shopify/manifest.json

```diff
@@ -0,0 +1,73 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "81c0f478-e722-454a-83d3-5e3f45e11ca8",
+  "app_id": "shopify",
+  "display_on_public_website": false,
```

Comment:
To confirm, the docs are only available in the integration tile and not on the public documentation site.

Reply:
Hey @estherk15, for all ntegrations we are building it has been decided that our PRs would always have this param value as False in the beginning, and when the integration is decided to be made GA for everyone, someone in DD team would turn the flag ON. You can reach out to @dj0well if there is an internal process for the same.

Reply:
Thank you for confirming @ashah-crest!